docs/hyperopt.md

```diff
@@ -1,153 +1,109 @@
 # Hyperopt
 This page explains how to tune your strategy by finding the optimal
-parameters with Hyperopt.
+parameters, process called hyperparameter optimization. The bot uses several 
+algorithms included in `scikit-optimize` package to accomplish this. The
+search will burn all your CPU cores, make your laptop sound like a fighter jet
+and still take a long time.
 
 ## Table of Contents
 - [Prepare your Hyperopt](#prepare-hyperopt)
-    - [1. Configure your Guards and Triggers](#1-configure-your-guards-and-triggers)
-    - [2. Update the hyperopt config file](#2-update-the-hyperopt-config-file)
-- [Advanced Hyperopt notions](#advanced-notions)
-    - [Understand the Guards and Triggers](#understand-the-guards-and-triggers)
+- [Configure your Guards and Triggers](#configure-your-guards-and-triggers)
+- [Solving a Mystery](#solving-a-mystery)
+- [Adding New Indicators](#adding-new-indicators)
 - [Execute Hyperopt](#execute-hyperopt)
 - [Understand the hyperopts result](#understand-the-backtesting-result)
 
-## Prepare Hyperopt
-Before we start digging in Hyperopt, we recommend you to take a look at 
-your strategy file located into [user_data/strategies/](https://github.com/freqtrade/freqtrade/blob/develop/user_data/strategies/test_strategy.py)
+## Prepare Hyperopting
+We recommend you start by taking a look at `hyperopt.py` file located in [freqtrade/optimize](https://github.com/freqtrade/freqtrade/blob/develop/freqtrade/optimize/hyperopt.py)
  
-### 1. Configure your Guards and Triggers
+### Configure your Guards and Triggers
 There are two places you need to change in your strategy file to add a 
 new buy strategy for testing:
```

Comment:
this is (currently) wrong - as pointed out in #566 ... the functions mentioned below are not in the strategy file.

Reply:
Fixed.